src/SFA.DAS.Payments.Core/Configuration/ConfigurationExtensions.cs

```
@@ -17,5 +17,10 @@ public static string GetSettingOrDefault(this IConfigurationHelper helper, strin
         {
             return helper.HasSetting("Settings", settingName) ? helper.GetSetting("Settings", settingName) : defaultValue;
         }
+
+        public static int GetSettingOrDefault(this IConfigurationHelper helper, string settingName, int defaultValue)
+        {
+            return helper.HasSetting("Settings", settingName) ? int.Parse(helper.GetSetting("Settings", settingName)) : defaultValue;
```

Use TryParse in case it's been misconfigured? Or does it make sense to fail fast?

---

I'd rather have it fail fast than wondering why is it ignoring a wrong value